ratatui/Cargo.toml

```
@@ -39,7 +39,7 @@ termwiz = ["dep:ratatui-termwiz"]
 #! The following optional features are available for all backends:
 ## enables serialization and deserialization of style and color types using the [`serde`] crate.
 ## This is useful if you want to save themes to a file.
-serde = ["dep:serde", "ratatui-core/serde", "ratatui-widgets/serde"]
+serde = ["dep:serde", "ratatui-core/serde", "ratatui-widgets/serde","ratatui-crossterm?/serde","ratatui-termion?/serde"]
```

nit: wrap this as it gets a bit long
```suggestion
serde = [
    "dep:serde",
    "ratatui-core/serde",
    "ratatui-widgets/serde",
    "ratatui-crossterm?/serde",
    "ratatui-termion?/serde"]
```